src/bp/ui-studio/src/web/views/FlowBuilder/common/routing.tsx

```diff
@@ -54,9 +56,12 @@ const getLabel = parsedCondition => {
 }
 
 export default class RoutingItem extends Component<Props> {
+  /* tslint:disable:prefer-function-over-method */
```

Comment:
rule has been remove here [here](https://github.com/botpress/botpress/pull/4105/files), can you copy/paste the fix